### DISCUSSION TITLE
Blocked account

### DISCUSSION THREAD

**Discussion by xvpc** [vercel/community#2505] [Help] [answered]
### Summary

Hello, I woke up today to see that my Vercel account was blocked for exceeding the Image Optimization limits, even tho I'm using a Hoppy account, as the image indicates. 
Now when I  navigate to any of my projects hosted on Vercel It shows me `402 PAYMENT_REQUIRED`.
Is there a way to fix this? , what should I do? Can I possibly disable Image Optimization?

### Additional information

Error message
![image](https://github.com/vercel/community/assets/36717692/4310dc5a-1afa-4ba7-a2e4-41aad2a2ce11)

Here It says that I have been blocked for reaching the maximum limits
![image](https://github.com/vercel/community/assets/36717692/9cff8215-0bf8-4bd7-8635-3ec2b74b50a0)

And here Image Optimization shows that it's more than 1,000 even though it should stop once It reaches the limit not (Block my account).
![image](https://github.com/vercel/community/assets/36717692/19f54cbb-156b-4b85-bc13-2f742c2322e1)

**Comment by amyegan** [comment, marked as answer]
I'm so sorry this happened, @xvpc. Since we'll need to discuss account details, it's best to [submit a case](https://vercel.com/help#issues) to the Customer Success Team (choose `Usage` as the Problem Area). They'll be able to help while keeping your private information out of this public discussion 🙇

**Reply by xvpc** [reply]
Alright, will do thank you.
